reference/components/cf-serverd.markdown

```diff
@@ -67,27 +60,31 @@ will potentially match more than one hostname (e.g.,
 **Examples**:
 
 ```cf3
-    allowallconnects      => {
+    allowconnects => {
          "127.0.0.1",
          "::1",
+         "200.1.10.0/24",
          "200\.1\.10\..*",
-         "host\.domain\.tld",
-         "host[0-9]+\.domain\.com"
          };
 ```
 
 
-### allowconnects
+### allowallconnects
 
-**Description:** List of IP addresses that may connect to the
-server port.
+**Description:** List of IP addresses that may have more than one
+connection to the server port. They are denoted in either IP or subnet
+form. For compatibility reasons, regular expressions are also accepted.
 
-If a client's identity matches an entry in this list it is granted
-to permission to send data to the server port. Clients who are not
-in this list may not connect or send data to the server.
+The clients that are not listed here may have only one open connection
+at the time with the server.
 
-See also the warning about regular expressions in
-[`allowallconnects`][cf-serverd#allowallconnects].
+Note that `127.0.0.1` is a regular expression (i.e., "127 any
+character 0 any character 0 any character 1"), but this will only
+match the IP address `127.0.0.1`. Take care with IP addresses and
+domain names, as the hostname regular expression `www.domain.com`
+will potentially match more than one hostname (e.g.,
```

Comment:
Yes you are right. This is a generic note for regexes (pre-existing, not added by me), but it should probably be under an attribute that accepts hostnames.